src/README.md

```diff
@@ -0,0 +1,136 @@
+# API setup:
+
+## Mongo Database setup:
```

Comment:
Do we all have to do this? Or is this just your documentation of what you did to set up a shared MongoDB instance that we can all use?

Reply:
That's documentation for "local setup". Unless u want people to mess with the database while testing their parts with the backend it's better for people to set up server locally, IMHO.